This patch originates from Debian:
https://anonscm.debian.org/cgit/debian-med/manta.git/tree/debian/patches/use_debian_packaged_samtools.patch

Author: Andreas Tille <tille@debian.org>
Last-Update: Tue, 15 Nov 2016 14:29:08 +0100
Description: Check for Debian packaged samtools

--- a/redist/CMakeLists.txt
+++ b/redist/CMakeLists.txt
@@ -63,34 +63,11 @@ superset(HTSLIB_LIBRARY "${CMAKE_INSTALL
 #
 # samtools
 #
-set(SAMTOOLS_PREFIX "samtools-1.2")
-set(SAMTOOLS_DIR "${CMAKE_CURRENT_BINARY_DIR}/${SAMTOOLS_PREFIX}")
-set(SAMTOOLS_LIBRARY "${SAMTOOLS_DIR}/libbam.a")
+set(SAMTOOLS_DIR "/usr/bin")
 superset(SAMTOOLS_PROG "${SAMTOOLS_DIR}/samtools")
 
-# final directory copy below would ideally be a soft-link, copy is for windows build
-add_custom_command(
-    OUTPUT ${SAMTOOLS_DIR}
-    COMMAND ${CMAKE_COMMAND} -E remove_directory "${SAMTOOLS_DIR}"
-    COMMAND ${CMAKE_COMMAND} -E tar xjf "${THIS_REDIST_DIR}/${SAMTOOLS_PREFIX}.tar.bz2"
-    COMMAND ${CMAKE_COMMAND} -E copy_directory "${HTSLIB_DIR}" "${SAMTOOLS_DIR}/${HTSLIB_PREFIX}"
-    DEPENDS ${HTSLIB_FINAL_TASK}
-    COMMENT "Unpacking samtools package")
-
-set (SAMTOOLS_FINAL_TASK ${SAMTOOLS_DIR})
+# set (SAMTOOLS_FINAL_TASK ${SAMTOOLS_DIR})
 
-if (NOT WIN32)
-    add_custom_command(
-        OUTPUT ${SAMTOOLS_PROG}
-        COMMAND $(MAKE) HTSDIR=${HTSLIB_PREFIX} all 2>| samtools.build.log
-        DEPENDS ${HTSLIB_LIBRARY}
-        DEPENDS ${SAMTOOLS_DIR}
-        WORKING_DIRECTORY ${SAMTOOLS_DIR}
-        COMMENT "Building samtools package")
-
-    set (SAMTOOLS_FINAL_TASK ${SAMTOOLS_PROG})
-endif ()
-
-set(THIS_SAMTOOLS "${THIS_PROJECT_NAME}_samtools")
-add_custom_target(${THIS_SAMTOOLS} DEPENDS "${SAMTOOLS_FINAL_TASK}")
+# set(THIS_SAMTOOLS "${THIS_PROJECT_NAME}_samtools")
+# add_custom_target(${THIS_SAMTOOLS} DEPENDS "${SAMTOOLS_FINAL_TASK}")
 
